website/docs/connect-data/how-to-guides/setup-datasource-environments.md

```diff
@@ -14,15 +14,45 @@ tags={[
 ]}
 />
 
+
+<Tags
+tags={[
+{ name: "Enterprise", link: "https://www.appsmith.com/pricing", additionalClass: "enterprise" }
+]}
+/>
+
+
 </div>
 
 <!-- vale on -->
 
 This page shows how to set up and manage multiple datasource environments for your application. For more details see [datasource environments](/connect-data/concepts/Datasource-Environments).
 
+
+## Create a custom environment
```

Comment:
why is this on top? also we are no longer using callouts for editions, but instead a tag.